src/main/java/CommandHub.java

```diff
@@ -0,0 +1,60 @@
+import java.util.HashMap;
+/*
+*  CommandHub is for creation and initialisation of all commands. Inclusive of setting up command specifics
+*   ie. setting up task to be removed
+*  "Parser"
+*
+* */
+
+
+public class CommandHub {
+    private static HashMap<String,UserCommand> commandList = new HashMap();
+
+    CommandHub(){
+    };
+
+    public static void initCreator(){
+        commandList.put("null",null);
+        commandList.put("bye",new Bye_Command());
+        commandList.put("list",new List_Command());
+    }
+
+    private static String[] parseString(String userInput){
+        userInput.toLowerCase();
+        String[] parsedCommand = userInput.split(" ", 2);
+        return parsedCommand;
+    }
+
+    public static UserCommand FetchCommand(String userInput) throws DukeException{
+        String parsedCommand[] = parseString(userInput);
+        String ID = parsedCommand[0];
+        String desc = "null";
+        if(parsedCommand.length>1) {//check if exist second line
+            desc = parsedCommand[1];
+        }
+
+        if(commandList.containsKey(ID)) { //gets generic commands
+            return commandList.get(ID);
+        }
+        else if(ID.contains("done")){ //done command
```

Comment:
The "else-if" statements should be on the same line as the preceding brace. Although, I also understand that it might get too cluttered 😅.